core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.ts

```diff
@@ -277,6 +277,10 @@ export class ConversationSkinComponent {
       this.skipButtonIsShown = true;
     }
 
+    if (!this.contextService.isExplorationPage()) {
```

Comment:
Can we use the existing `isInExplorationPlayerPage` method in the context service?

Reply:
Done :)